com.microsoft.java.debug.plugin/src/main/java/com/microsoft/java/debug/plugin/internal/eval/JdtEvaluationProvider.java

```diff
@@ -178,6 +188,37 @@ public void clearState(ThreadReference thread) {
         }
     }
 
+    private boolean ensureDebugTarget(VirtualMachine vm) {
```

Comment:
The return value of ensureDebugTarget is never referenced anywhere. In the implementation, you use both exception and return value to indicate success, which is not recommended.

'ensure' means make sure something is actually there. When the function is not able to ensure that, an exception should be thrown. So please only use exceptions and the return value should be void.

Reply:
sure